docker-build/entrypoint.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
+if [ -f "/opt/conda/etc/profile.d/conda.sh" ]; then
+    . "/opt/conda/etc/profile.d/conda.sh"
+    CONDA_CHANGEPS1=false conda activate invokeai
+fi
 
-cd /stable-diffusion
+#cd /InvokeAI/
+conda activate invokeai
 
-if [ $# -eq 0 ]; then
```

Comment:
I like to have the option for my own parameters, which was possible here and is not available in your entrypoint

Reply:
I can work on re-adding that, major goal was just getting something that worked.